deps/build.jl

```diff
@@ -0,0 +1,64 @@
+# remove deps.jl if it exists, in case build.jl fails
+isfile("deps.jl") && rm("deps.jl")
+
+@unix_only const LIBPARDISONAMES = [
+    "libpardiso500-GNU461-X86-64",
+    "libpardiso500-GNU472-X86-64",
+    "libpardiso500-GNU481-X86-64",
+    "libpardiso"
+]
+
+@windows_only const LIBPARDISONAMES = ["libpardiso500-WIN-X86-64.dll", "libpardiso"] 
+
+const PATH_PREFIXES = [
+    joinpath(Pkg.dir(), "Pardiso", "deps"),
```

Comment:
don't use Pkg.dir, use `dirname(@__FILE__)` so the package can be installed elsewhere